src/azure-cli/HISTORY.rst

```diff
@@ -16,6 +16,7 @@ Release History
 **ARM**
 
 * Update azure-mgmt-resource package to use 6.0.0
+* az policy set-definition create/update: Add support for `--metadata`.
```

Comment:
`Add new parameter `--metadata` to support adding metadata information for policy set definitions` may be more clear.

Reply:
Resolved